Lib/test/test_inspect.py

```diff
@@ -2321,6 +2321,138 @@ async def func(a=None):
                          {'a': None, 'gencoro': gencoro, 'b': 'spam'})
 
 
+class TestGetAsyncGenState(unittest.TestCase):
```

Comment:
I suggest to inherit from IsolatedAsyncioTestCase and use `await` with `anext` rather than `next`.